Tella/Scenes/Settings/ViewModel/UwaziServerViewModel.swift

```diff
@@ -204,7 +203,7 @@ class UwaziServerViewModel: ObservableObject {
                     print("Finished")
```

Comment:
Could we remove this print? Or maybe try to change it for a debugLog

Reply:
Changed with debug log and other print statement also changed with debug log